rllib/utils/numpy.py

```diff
@@ -0,0 +1,198 @@
+from __future__ import absolute_import
+from __future__ import division
```

Comment:
What's this for?

Reply:
Test cases: Numpy-based checks vs simple tf/torch model forward passes for e.g. loss function tests.

Reply:
e.g. you create a Model with a very simple NN (1 fc layer w/ 10 linear nodes). Then simulate the forward pass, check the Model's output against its weights using this module's `fc()`-function and then calculate the exact expected loss value and  validate it with the loss functions actual output.

Reply:
Note: The next PR will have test cases for PG- and PPO losses (both torch and tf).

Reply:
Hm looks fine to merge, though isn't it a bit like testing TF itself?